packages/neos-ui/src/Containers/PrimaryToolbar/MenuToggler/style.css

```diff
@@ -1,16 +1,31 @@
 .menuToggler {
+    width: calc(var(--spacing-GoldenUnit) * 4);
+
     &:focus {
         outline: 0;
     }
 }
 
 .menuToggler--isActive {
     color: var(--colors-PrimaryBlue);
+    background: var(--colors-PrimaryBlue);
+    color: #fff;
+
+    .menuToggler__icon {
+        color: #fff;
+
+        &:hover,
+        &:focus {
+            color: #fff;
```

Comment:
You can use the constants ;) and the new color makes no sense when it is the same.

color: var(--contrastBrightest);

Reply:
Sorry, obviously missed that (happened during merge ...). Will fix of course.